src/features/pilot_management/PilotSheet/sections/mech/components/MechNav.vue

```diff
@@ -42,7 +48,9 @@
     <v-menu offset-y top>
       <template v-slot:activator="{ on: menu }">
         <v-btn class="unskew ml-2" icon dark v-on="menu">
-          <v-icon>mdi-settings</v-icon>
+          <cc-tooltip inline delayed content="Mech Options">
```

Comment:
Conversely, I did find that I can't move the cc-tooltip element any higher for the two menu buttons, as it causes the menu to render in the upper left corner of the screen when you click the button.